### ISSUE TITLE
How to adjust the percentage of the training data with labels for NTU RGB+60？

### ISSUE THREAD

**Issue by ch1998** [fmthoker/skeleton-contrast#1]
Your code is wonderful。
In the paper，you report results whenusing 1%, 5%, 10% and 20% of the training data with labels for NTU RGB+60。
I hope to how adjust the percentage of the training data?

**Comment by fmthoker** [comment]
@ch1998  you can randomly select any 5% of training examples during the fine-tuning step.  However for reproducibility seed the random generator.

**Comment by ch1998** [comment]
I get it. Thank you!
